tests/unit/test_dns_client.cpp

```diff
@@ -135,6 +135,9 @@ void otPlatFree(void *aPtr)
 
     free(aPtr);
 }
+
+void *otPlatCryptoCAlloc(size_t aNum, size_t aSize) { return calloc(aNum, aSize); }
+void  otPlatCryptoFree(void *aPtr) { free(aPtr); }
```

Comment:
Can we define these in the `tests/unit/test_platform.cpp`?
This way it will be available for all unit tests to use?